src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -141,7 +145,12 @@ public async Task<IActionResult> OnGetAsync(string id, string revisionId = null)
             {
                 foreach (var username in approverConfig.Split(","))
                 {
-                    approvers.Add(username);
+                    UserProfileModel user = await _userProfileRepository.tryGetUserProfileByNameAsync(username);
```

Comment:
We are querying from cosmos for every review. This is something we should store in user preference cache to avoid fetching from DB when user opens a review. It's not a blocker for this PR to merge but a good to do change.

Reply:
@praveenkuttappan have changed so that for the current user, the cache is checked first for languages and if none are found, the UserProfileModel is retrieved from the database and then languages are saved to the cache. For all other users, I have left as is, but do you think it's worth doing the same for them?

Reply:
I think this change is fine.